subprojects/docs/src/docs/userguide/depMngmt.adoc

```diff
@@ -544,6 +547,18 @@ Now Gradle will look for your dependencies in the JCenter repository. _jcenter()
             </sample>
 ++++
 
+[[sub:maven_google]]
+==== Maven Google repository
```

Comment:
We should add a link to Android's docs here too: https://developer.android.com/studio/build/dependencies.html#google-maven